src/main/java/com/google/cloud/teleport/templates/common/SplunkConverters.java

```diff
@@ -116,6 +116,13 @@ public static FailsafeStringToSplunkEvent failsafeStringToSplunkEvent(
     ValueProvider<Boolean> getIncludePubsubMessage();
   
     void setIncludePubsubMessage(ValueProvider<Boolean> includePubsubMessage);
+
+    @Description(
+        "KMS Encryption Key should be in the format "
```

Comment:
specify that this is a kms encryption key for the token.

Reply:
Done